index.php

```diff
@@ -0,0 +1,169 @@
+    <!-- INDEX.PHP  -->
+    <?php
+    require 'imageForm.php';
+
+    // creating a new 'Form' object
+    $myForm = new Form();
+
+    // at this point all the properties are set to default values by constructor
+
+    if ($_SERVER["REQUEST_METHOD"] == "POST") {
+
+        if (empty($_POST["fname"])) {
```

Comment:
Could we merge this two `if` statement?

Reply:
Sorry, but I didn't get this. On line 10, I have check whether the form has been submitted whereas on line 12 and henceforth(line 24, line 38 and others), I have checked for the values of different input fields when the form is submitted.